### PR TITLE
Uml 1302 : content changes on activation key sent page

### PR DESCRIPTION
**Pull request by SeemaMenon**
# Purpose

https://opgtransform.atlassian.net/browse/UML-1302

Fixes UML-1302

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [x] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/app/features/context/UI/LpaContext.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ public function aLetterIsRequestedContainingAOneTimeUseCode()
     {
         $this->ui->assertPageAddress('/lpa/check-answers');
 
-        $this->ui->assertElementContainsText('h1', 'We\'re sending you an activation key');
+        $this->ui->assertElementContainsText('h1', 'We’re posting you an activation key');
```
Comment (GemTay) [inline service-front/app/features/context/UI/LpaContext.php:52]: The apostrophe needs to still be escaped with the backslash, also what character have you replaced the apostrophe with? It looks different? Probably best to change it back to the standard one

File: service-front/app/languages/cy/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -5544,3 +5543,34 @@ msgstr "Nodwch eich enw olaf"
...
+
+#: src/Actor/templates/actor/send-activation-key-confirmation.html.twig:25
+msgid "We've sent you an email to confirm all these details."
+msgstr "Rydym wedi anfon e-bost atoch yn cadarnhau’r holl fanylion hyn."
+
+#: src/Actor/templates/actor/send-activation-key-confirmation.html.twig:26
+msgid "You should get the letter by %date%"
+msgstr "Dylech gael y llythyr erbyn %date%"
+
+# Need to switch OPG & LPA in the translation
```
Comment (GemTay) [inline service-front/app/languages/cy/LC_MESSAGES/messages.po:5569]: does this comment need to be kept here?

File: service-front/app/src/Actor/templates/actor/send-activation-key-confirmation.html.twig [hunk 3]
```diff
@@ -15,22 +15,20 @@
                 <div class="govuk-grid-column-full">
                     <div class="govuk-panel govuk-panel--confirmation">
                         <h1 class="govuk-panel__title">
-                            {% trans %}We're sending you an activation key{% endtrans %}
+                            {% trans %}We’re posting you an activation key{% endtrans %}
```
Comment (GemTay) [inline service-front/app/src/Actor/templates/actor/send-activation-key-confirmation.html.twig:18]: strange apostrophe again

**Review by GemTay — CHANGES_REQUESTED** [review]
just a few queries

File: service-front/app/languages/cy/LC_MESSAGES/messages.po (hunk 2 above)
Comment (SeemaMenon) [reply service-front/app/languages/cy/LC_MESSAGES/messages.po:5569]: Am not sure about this? Not introduced by me...

File: service-front/app/src/Actor/templates/actor/send-activation-key-confirmation.html.twig (hunk 3 above)
Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/send-activation-key-confirmation.html.twig:18]: Wonder how it keeps changing. :(

File: service-front/app/features/context/UI/LpaContext.php (hunk 1 above)
Comment (SeemaMenon) [reply service-front/app/features/context/UI/LpaContext.php]: done

**Review by GemTay — APPROVED** [review]

**Pull request merged**